sql-parser/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/MySQLStatementVisitor.java

```diff
@@ -474,12 +481,17 @@ private ASTNode createAssignmentSegment(final BooleanPrimaryContext ctx) {
     private ASTNode createCompareSegment(final BooleanPrimaryContext ctx) {
         ExpressionSegment left = (ExpressionSegment) visit(ctx.booleanPrimary());
         ExpressionSegment right;
+        String operator;
+        if (null != ctx.ALL()) {
```

Comment:
It seems that this parse logic change has impacted encrypt like feature.

Reply:
Hi @kanha-gupta, I test your pr, you can try add `SUPPORTED_COMPARE_OPERATOR.add("IS NULL");
        SUPPORTED_COMPARE_OPERATOR.add("IS NOT NULL");` to EncryptConditionEngine, this exception can be solved.

Reply:
Thanks a lot for guidance sir :)